src/lib.rs

```diff
@@ -573,6 +574,51 @@ impl<T: ArtifactOutput> Project<T> {
         Ok(input)
     }
 
+    /// Returns standard-json-input to compile the target contract
+    pub fn zk_standard_json_input(
```

Comment:
```suggestion
    pub fn zksync_standard_json_input(
```
We used zksync prefix on this repo. Although it doesn't matter that much as this whole api changes on further versions